samples/linux/jobs_sample/aws_iot_config.h

```diff
@@ -70,4 +70,9 @@
 
 #define DISABLE_METRICS false ///< Disable the collection of metrics by setting this to true
 
+// TLS configs
+#define IOT_SSL_READ_TIMEOUT 10 ///< Timeout associated with underlying socket of TLS connection (set by mbedtls_ssl_conf_read_timeout)
+#define IOT_SSL_READ_RETRY_TIMEOUT_MS 10 ///< Minimum elapsed time before returning from iot_tls_read when pending data has not yet been received
+#define IOT_SSL_WRITE_RETRY_TIMEOUT_MS 10 ///< Minimum elapsed time before returning from iot_tls_write when pending data has not yet been written
```

Comment:
Does giving the same value to the retry-timeouts as the IOT_SSL_READ_TIMEOUT attempt a retry, incase the ssl_read times out? Doesn't it need to be higher than IOT_SSL_READ_TIMEOUT to retry at least once in case of a ssl read or write timeout?

Reply:
Sure, changing `IOT_SSL_READ_TIMEOUT` to `3`, allowing 4 maximum tries.